### PR TITLE
Add building commands

### PR DESCRIPTION
**Pull request by lalabuy948**
Just few crossplatform building commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,7 +267,20 @@ To dump the latest 100 users with their orders:
       created_at = "desc"
 ```
 
+## Build commands
 
+For linux
+
+```zsh
+GOOS=linux GOARCH=amd64 go build -o klepto-linux
```
Comment (usoban) [inline README.md]: `klepto-linux-amd64` to indicate target architecture. Because you might be running Linux on ARM, but this binary would not work there

**Review by usoban — CHANGES_REQUESTED** [review]

**Comment by usoban** [comment]
Beautiful :)

File: README.md (hunk 1 above)
Comment (lalabuy948) [reply README.md]: Yeah, i will add it as well

Comment (lalabuy948) [reply README.md]: I will add it only for linux, coz darwin runs only on amd64

**Review by usoban — APPROVED** [review]

**Comment by usoban** [comment]
I needed to add `CGO_ENABLED=0` env variable, otherwise the binary did not run even on other linux machines.

**Pull request merged**